substrate/client/authority-discovery/src/worker.rs

```diff
@@ -539,15 +567,13 @@ where
 				}
 
 				if log_enabled!(log::Level::Debug) {
-					let hashes: Vec<_> = v.iter().map(|(hash, _value)| hash.clone()).collect();
-					debug!(target: LOG_TARGET, "Value for hash '{:?}' found on Dht.", hashes);
+					debug!(target: LOG_TARGET, "Value for hash '{:?}' found on Dht.", v.record.key);
 				}
```

Comment:
Since we don't allocate anymore here:

```suggestion

				debug!(target: LOG_TARGET, "Value for hash '{:?}' found on Dht.", v.record.key);

```